src/renderer/components/ImageCropper/index.tsx

```diff
@@ -289,9 +360,40 @@ export default function ImageCropper({
               src={filepath}
             />
           </div>
+          <div className={styles.imageCropperControls}>
+            <button
+              className={styles.imageCropperControlsButton}
+              onClick={onZoomIn}
+              aria-label={tx('menu_zoom_in')}
+            >
+              <Icon coloring='navbar' icon='plus' size={20} />
+            </button>
+            <button
+              className={styles.imageCropperControlsButton}
+              onClick={onZoomOut}
+              aria-label={tx('menu_zoom_out')}
+            >
+              <Icon coloring='navbar' icon='minus' size={24} />
+            </button>
+            <button
+              className={styles.imageCropperControlsButton}
+              onClick={onRotateImages}
+              aria-label={tx('ImageEditorHud_rotate')}
+            >
+              <Icon coloring='navbar' icon='rotate-right' size={24} />
+            </button>
+            <button
+              className={styles.imageCropperControlsButton}
+              onClick={onZoomReset}
+              aria-label={tx('reset')}
```

Comment:
How about creating a new component for all of these?

Reply:
Not a big fan of decomposition for the sake of decomposition. I think it's much simple to understand what's happening in the code, doesn't introduce a lot of repetitive function names (when we pass them to this new component via props) when you search text in a project.
I treat it like a "send" button in a message dialog, just UI to control the component.